app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -82,7 +82,7 @@ class Identification extends React.Component {
         // If date is undefined, age will stay undefined (which nulls out the age field)
         if (dateOfBirth) {
           age = 0 - moment(dateOfBirth).diff(moment.now(), 'years');
-          age = age < 200 && age > 0 ? age : self.state.current.patient.age;
+          age = age > 0 ? age : self.state.current.patient.age;
```

Comment:
There is a bug where If you clear out the date of birth and then try and set the date of birth to another value, the age won't repopulate. This appears to be because this line is setting age to be 0: `age = 0 - moment(dateOfBirth).diff(moment.now(), 'years');`

Reply:
fixed here: https://github.com/SaraAlert/SaraAlert/pull/390/commits/693a8594546922023eb75c58228da28ebd7c6e65